_articles/getting-started/import-from-lastpass.md

```diff
@@ -21,6 +21,11 @@ Importing your data from LastPass into bitwarden is easy.
 
 **Export Your Passwords From The LastPass Browser Extension**
 
+```
+!!!!!! IMPORTANT !!!!!!
+If you are using the Chrome extension, export through the web vault instead (above)
```

Comment:
Let's add a little more detail like:

`If you are using the LastPass Chrome extension, it is preferred to export through the LastPass web vault in order to avoid known bugs with the extension exporter.`